Makefile

```diff
@@ -248,35 +248,26 @@ build-container-local:
 .PHONY: build-container-dev
 build-container-dev:
 	$(MAKE) _build \
-		IMG="$(IMAGE_NAME)-dev" \
-		DOCKERFILE="Dockerfile_dev" \
+		IMG="$(IMAGE_NAME_DEV)" \
 		TAG="$(BUILD_TAG)"
 
 #-------------------------------------------------------------------------------
 
 # dev
 
-.PHONY: _build-container-multiplatform-dev
-_build-container-multiplatform-dev:
-	BUILD_TAG="$(BUILD_TAG)" \
-		REPO_URL="$(OPENSOURCE_REPO_URL_DEV)" \
-		DOCKERFILE="Dockerfile_dev" \
-		PLATFORM="$(PLATFORM)" \
-		./ci/build-push-multiplatform.sh $(PUSH)
-
 .PHONY: build-container-multiplatform-dev
-build-container-multiplatform-dev:
-	$(MAKE) _build-container-multiplatform-dev PUSH=
+build-container-multiplatform-dev: REPO_URL="$(OPENSOURCE_REPO_URL)"
+build-container-multiplatform-dev: build-container-multiplatform
```

Comment:
Does it work as expected :open_mouth: ?

Reply:
No, but it's pretty close. I set `REPO_URL` to `OPENSOURCE_REPO_URL` by default, and then used the former in the release targets.

Reply:
so it works as
```
build-container-multiplatform-dev: REPO_URL="$(OPENSOURCE_REPO_URL)" build-container-multiplatform
```
?

Reply:
Works more or less like:
```
build-container-multiplatform-dev:  
  make build-container-multiplatform REPO_URL="$(OPENSOURCE_REPO_URL)"
```

except it doesn't start a new shell, and you can do more stuff in the variable assignment.